src/groupederror.jl

```diff
@@ -202,14 +204,24 @@ Output is a `GroupedError` with error computed according to the keyword `compute
 It can be plotted using `plot(g::GroupedError)`
 Seriestype can be specified to be `:path`, `:scatter` or `:bar`
 """
-function groupapply(f::Function, df, args...;
-                    axis_type = :auto, compute_error = :none, group = [],
+function groupapply(f::Function, dfn, args...;
+                    axis_type = :auto, compute_error = :none, group = Array{Symbol,1}(0),
                     summarize = (get_symbol(compute_error) == :bootstrap) ? (mean, std) : (mean, sem),
                     compute_axis = :separate,
                     kwargs...)
+
     if !(axis_type in [:discrete, :continuous])
-        axis_type = (typeof(df[args[1]])<:PooledDataArray) ? :discrete : :continuous
+        axis_type = (typeof(dfn[args[1]])<:Union{CategoricalArray,NullableCategoricalArray}) ?
```

Comment:
Yes, I just had the same thought in response to the comment here: https://github.com/JuliaPlots/StatPlots.jl/pull/34#issuecomment-274056998
In StatPlots, both `Array{String}` and `Array{Symbol}` are automatically treated as categorical variables.